rebar.lock

```diff
@@ -0,0 +1,178 @@
+{"1.1.0",
```

Comment:
rebar.lock? Definitely, it makes sure all deps are frozen, even if a dep of dep uses master branch.